renogyapi/__init__.py

```
@@ -69,49 +71,62 @@ async def process_request(
                         message = {"error": message}
-                    return message
+                return message
 
-            except (TimeoutError, ServerTimeoutError):
-                _LOGGER.error("%s: %s", ERROR_TIMEOUT, url)
-                message = {"error": ERROR_TIMEOUT}
-            except ContentTypeError as err:
-                _LOGGER.error("%s", err)
-                message = {"error": err}
+        except (TimeoutError, ServerTimeoutError):
+            _LOGGER.error("%s: %s", ERROR_TIMEOUT, url)
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)